### ISSUE TITLE
Data Futures Lab landscape analysis: Create PDFs

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4829]
**Project expected SOFT launch:** Web Sept 3rd < Taís' edit

Eeva will move the content to our templates then Tais will do a design pass. Mood (updated on Aug 27th): light, a bit fun. Not too scientific but also not super fun and light as IHR.

### Deliverables
- [ ] Design review of PDF of Foundations research (big ~60 page publication) ([draft](https://docs.google.com/document/d/1BBJRmIh0krDSy7djGFQFORuVNgN7k3PMzOC0R0bmTlE/edit#heading=h.mefjv4rjbj5d)) 
- [ ] Design review of PDF of History of data stewardship (medium ~10 page publication) ([outline](https://docs.google.com/document/d/1sDFJ6iUR7afgU9QfW_W7J7-8Uwp7E14BSfIVqgqfJ_U/edit)) (need draft)
- [ ] Design Review of PDF of Database analysis (small ~20 page publication) ([draft](https://docs.google.com/document/d/17DXwX-DfzXwawoe3lwriNHHBCbtyH-7PoRCTowyZr3E/edit#heading=h.q57oq65fh8g9))
- [ ] On brand AirTable of database - [Airtable here](https://airtable.com/tbl6wGsRuAebCHqlz/viwqVwABuZqSFxJ39?blocks=hide)
- AirTable needs one more illustration for the most recent item added "Submit a Project"
- [ ] All graphs/charts are styled and on brand
- [ ] Color-coded cover (probably an pattern/illustration) to add to each one of the reports.

**Comment by kristinashu** [comment]
Moving this to the next sprint since content won't be final until the end of the month.

**Comment by taisdesouzalessa** [comment]
Pausing this project during this sprint (July 21-Aug03): waiting for the Insights team to move on this project a bit more before we get into the design stage. We will connect with that team next sprint and reassess if it is ready for design.

**Latest references:**

- [Online Presence Brief and Mockup](https://docs.google.com/document/d/1X1kAvb0-e7zzYS-eXFiTns6lXQmstxVI7GtWuH3BhPA/edit?usp=sharing)
- [Draft 2020 Publications](https://docs.google.com/document/d/1jTQHYwgHjym8jU50o8P-94droNe54mD1-7mPB35HBJ4/edit?usp=sharing)
- Email titled _"Data Stewardship landscape analysis online plan"_

**Comment by taisdesouzalessa** [comment]
tagging you here too @mehan

**Comment by taisdesouzalessa** [comment]
Blocked by getting more info from Insights team. We will have a meeting on **Aug 27th (Thursday)** that may unblock this work.

**Comment by taisdesouzalessa** [comment]
Eeva and Kasia will populate the PDFs and we will give it a design pass.
What we will need for each project (**nice to have**) is a color coded cover (maybe just a pattern/quick illustration) so we can use the same color code on the different site sections.

**To do:**
- [ ] color coded illustrations/patterns for 3 reports.

This could be done over next week (Aug 31)

**Comment by taisdesouzalessa** [comment]
Deadline changed to Sept 4th. We don't have the templates populated yet by the Insights team. Created a new ticket for this task for next print: #5164
